specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/networkManager.json

```diff
@@ -437,6 +437,102 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/networkManagers/{networkManagerName}/inviteMultiTenantScope": {
+      "post": {
+        "tags": [
+          "NetworkManagers"
+        ],
+        "operationId": "MultiTenantScopes_Invite",
+        "description": "Issue an invitation to a scope of another tenant to be managed by this Network Manager.",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/NetworkManagerNameParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/MultiTenantScopeRecord"
+            },
+            "description": "Multi Tenant Scope record."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns a list of effective virtual networks.",
+            "schema": {
+              "$ref": "#/definitions/MultiTenantScopeRecord"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Invite secondary tenant scope to be managed by Network Manager": {
+            "$ref": "./examples/NetworkManagerMultiTenantScopeInvite.json"
+          }
+        }
+      }
+    },
+    "{scope}/providers/Microsoft.Network/acceptMultiTenantScopeInvitation": {
+      "post": {
+        "tags": [
+          "NetworkManagers"
+        ],
+        "operationId": "MultiTenantScopes_AcceptInvite",
+        "description": "Accept an invitation as a scope of another tenant to be managed by this Network Manager.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ScopeParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/MultiTenantScopeRecord"
```

Comment:
>MultiTenantScopeRecord [](http://example.com/codeflow?start=37&length=22)

Where does the ID of the network manager go in this definition?

Reply:
The scope record has the following properties,
1. Network Manager Id
2. Scope Id
3. Scope Tenant
4. Provisioning status